src/librawspeed/metadata/CameraMetaData.h

```diff
@@ -23,18 +23,21 @@
 #include "common/Common.h" // for uint32
 #include <map>             // for map
 #include <string>          // for string
+#include <tuple>           // for tuple
 
 namespace RawSpeed {
 
 class Camera;
 
+using CameraId = std::tuple<std::string, std::string, std::string>;
+
```

Comment:
Also related to https://github.com/darktable-org/rawspeed/pull/61/commits/049d0ebe5cf20f9deabb9063fcf2e5061c0ff7eb